### DISCUSSION TITLE
How do you make the number 1️⃣ disappear?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1098] [Galaxy Trivia] [answered]
**Bonus:** What two 🔑🗝️ can’t open any 🚪?

**Comment by ParanoidUser** [comment]
Hey, riddle raiders (@Heidet / @hammadalive7 / @Spixmacaw)! Up for a brain teaser? 🤓

**Reply by hammadalive7** [reply, marked as answer]
1) By adding 'G' Infront of 1 will make it's "GONE" XD
2) Monkey and Donkey

**Comment by peterbaccaro** [comment]
You can make the number “one” disappear by **adding the letter “G” to it and it is “GONE”**.

Two keys that can't open any doors are Don**key** and Mon**key**.

**Comment by thuongtruong1009** [comment]
To make the number 1️⃣ disappear, you can simply cover or erase it.

The answer to the second riddle, "What two 🔑🗝️ can’t open any 🚪?" is "A monkey and a donkey." These are not typical keys and cannot open doors like regular keys can.
